### PR TITLE
CUMULUS-295 logging and schema

### PR DESCRIPTION
**Pull request by sethvincent**
In progress changes for CUMULUS-295.

- Updated to use latest cumulus-message-adapter
- Passes `schemas` object to cumulus-message-adapter
- Logs exceptions using the CumulusLogger from cumulus-message-adapter

### CODE REVIEW

File: run_cumulus_task.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import os
 import sys
 
+from cumulus_logger import CumulusLogger
```
Comment (scisco) [inline run_cumulus_task.py]: Is this how others tasks should use this logger too?

Comment (sethvincent) [reply run_cumulus_task.py]: yes

File: README.md [hunk 2]
```diff
@@ -54,13 +54,14 @@ must be communicated to those using the task.
...
 The Cumulus Message adapter for python provides one method:
-`run_cumulus_task`. It takes three parameters:
+`run_cumulus_task`. It takes four parameters:
 
   * `task_function` - the function containing your business logic (as described
     above)
   * `cumulus_message` - the event passed by Lambda, and should be a Cumulus
     Message
   * `context` - the Lambda context
+  * `schemas` - optional: a dict with `input`, `config`, and `output` properties. Each should be set to the filepath of the corresponding JSON schema file
```
Comment (laurenfrederick) [inline README.md]: Might want to say all 3 are not required

File: cumulus_logger.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    def createMessage(self, message):
+        if type(message) is str:
+            message = {
+                "message": message
+            }
+
+        try:
+            message["level"]
+        except KeyError:
+            message["level"] = "info"
```
Comment (laurenfrederick) [inline cumulus_logger.py]: Something we did in the other loggers is create a function for each level, that way the user of the `log()` function doesn't have to know about the levels, they can call a separate function for error, warning, etc. Then you're not dependent on someone knowing the correct string to use, so I think that would be a good idea here.

File: README.md (hunk 2 above)
Comment (laurenfrederick) [reply README.md]: Oh and add something to the README about how to do logging

**Pull request merged**